pallets/liquidity-pools-gateway/routers/src/lib.rs

```diff
@@ -197,19 +210,7 @@ where
 	/// Sets the weight information for the provided XCM domain location, and
 	/// the fee per second for the provided fee asset location.
 	pub fn do_init(&self) -> DispatchResult {
-		pallet_xcm_transactor::Pallet::<T>::set_transact_info(
```

Comment:
Did we test to confirm that none of these are needed or do we manually need to set these storages after creating a router?

Reply:
Non are needed. The `pallet-xcm-transactor` can work with user provided values and only fills in the `Option` weights and fees if they are not provided with the defaults.

Reply:
I tested that I still sends with `Okay`

Reply:
There is a test for that I mean